apps/web/screens/UniversitySearch/LandingPage.tsx

```diff
@@ -0,0 +1,218 @@
+import { useState } from 'react'
+import cn from 'classnames'
+import { useRouter } from 'next/router'
+
+import { Box, GridColumn, Icon, Input } from '@island.is/island-ui/core'
+import {
+  getThemeConfig,
+  OrganizationWrapper,
+  SliceMachine,
+} from '@island.is/web/components'
+import {
+  ContentLanguage,
+  Query,
+  QueryGetNamespaceArgs,
+  QueryGetOrganizationPageArgs,
+} from '@island.is/web/graphql/schema'
+import { useLinkResolver, useNamespace } from '@island.is/web/hooks'
+import { withMainLayout } from '@island.is/web/layouts/main'
+import { CustomNextError } from '@island.is/web/units/errors'
+
+import { Screen } from '../../types'
+import { LandingPageFooter } from '../Organization/Home/LandingPage'
+import {
+  GET_NAMESPACE_QUERY,
+  GET_ORGANIZATION_PAGE_QUERY,
+  GET_ORGANIZATION_QUERY,
+} from '../queries'
+import * as styles from './UniversitySearch.css'
+interface LandingPageProps {
+  organizationPage?: Query['getOrganizationPage']
+  organization?: Query['getOrganization']
+  namespace: Query['getNamespace']
+  locale: string
+}
+const LandingPage: Screen<LandingPageProps> = ({
+  organizationPage,
+  namespace,
+}) => {
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore make web strict
```

Comment:
You could change the namespace type in the LandingPageProps interface to be a Record<string, string> and then you wouldn't need to add a ts-ignore here